### DISCUSSION TITLE
Static class members should be static by default?

### DISCUSSION THREAD

**Discussion by nextgenadarsh** [dotnet/csharplang#8440] [Polls]
Sample code:

public class MyStaticClass {
  public static string Property1 { get; set; }
  public string Property2 { get; set; } // compiler error
}

**Comment by Joe4evr** [comment]
This is a very intentional design decision of the language. If you started out designing a static class, then in a later version made it non-static, without the current restriction all your members would now be instance members and all your callsites are broken.
